core/execution/market_for_test.go

```diff
@@ -43,13 +43,14 @@ func (m *Market) GetPeggedOrderCount() int {
 
 // GetParkedOrderCount returns hte number of parked orders in the market.
 func (m *Market) GetParkedOrderCount() int {
-	var count int
-	for _, order := range m.peggedOrders.orders {
-		if order.Status == types.OrderStatusParked {
-			count++
-		}
-	}
-	return count
+	// var count int
```

Comment:
shouldn't we just outright delete this code?

Reply:
yes we should

Reply:
done